src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.js

```diff
@@ -42,27 +40,7 @@ function extractAttachmentsFromReport(report, reportActions, source) {
     });
     htmlParser.end();
 
-    attachments = attachments.reverse();
-
-    const initialPage = _.findIndex(attachments, (a) => a.source === source);
```

Comment:
Why couldn't we have this logic here?

Reply:
I need only to move `Navigation.dismissModal()` out of this function, but other PRS renamed the file `createInitialState` to `extractAttachmentsFromReport`, so I think it will be more clean if this function return only attachments array to be suitable for its name, but we can keep them.